### ISSUE TITLE
question

### ISSUE THREAD

**Issue by zc994646659** [JaGeo/LobsterPy#24]
**Describe the bug**
when i run lobsterpy automatic-plot --allbonds
it reported 
/data/software/anaconda/2020.07/envs/lobsterpy/lib/python3.8/site-packages/pymatgen/electronic_structure/cohp.py:164: UserWarning: This method has not been tested on many examples. Check the parameter limit, pls!
  warnings.warn("This method has not been tested on many examples. Check the parameter limit, pls!")
The compound CrN has 1 symmetry-independent atoms(s) with relevant bonds: Cr1.
Cr1 has a single (CN=1) coordination environment. It has 1 Cr-Cr (mean ICOHP: -0.2 eV, antibonding interaction below EFermi) bonds.

what does it mean ? the Cr-Cr is anti bonding? but i cannot seen it in my picture. And it did not generate any output? How can I do with it.I asked before, but I didn't check the reply in time. I'm very sorry to ask again

**Comment by JaGeo** [comment]
Thanks for your question and thank you for using LobsterPy.

No, it just means that some antibonding states below Efermi are occupied. It does not have to be much. Are more detailed analysis will be available in a later version of LobsterPy. 

And: what do you mean by "it did not generate any output"? What kind of output are you expecting? We print the results to the screen and plot them.

**Comment by JaGeo** [comment]
Thank you for your more detailed answer. Unfortunately, the zip archive seems to be broken. Could you send it again?

Thanks.

**Comment by zc994646659** [comment]
sorry
[CrN.zip](https://github.com/JaGeo/LobsterPy/files/8447840/CrN.zip)
[WN.zip](https://github.com/JaGeo/LobsterPy/files/8447841/WN.zip)
thank you for your help

**Comment by JaGeo** [comment]
I have looked at your files: 
- the problem is that lobsterpy can only analyze bonds that are included in the COHP computation.
- currently, you only have one bond per ICOHPLIST.lobster. Thus, lobsterpy will only detect this. You would need `lobsterpy automaticplot --allbonds` to analyze your Cr-Cr bond. This is of course a very weak bond and the COHP plot includes some occupied antibonding states. 
- To use lobsterpy, you should use the COHPGenerator in the lobsterin file. A corresponding lobsterin file could look like this:

`COHPstartEnergy -35.0 

COHPendEnergy 5.0 

basisSet pbeVaspFit2015 

cohpGenerator from 0.1 to 6.0 orbitalwise 

saveProjectionToFile 

basisfunctions Cr 3d 3p 4s 

basisfunctions N 2p 2s
`
`lobsterpy create-inputs` can suggest such inputs. Please check these files if they make sense. Please also always test your projection. Have a look at the charge spilling and test if your results fit your expectations. This command sometimes also creates more than one lobsterin (lobsterin.lobsterpy-0). It will also adapt INCARs (NBANDS, LWAVE=True, ISYM=0, NSW=0) automatically and create INCAR.lobsterpy-0 and so on.

- lobsterpy can only analyze what it is in your files. :)
- I hope this helps!

**Comment by JaGeo** [comment]
We will write a tutorial to make this more obvious! Will ping you once it is finished.

**Comment by JaGeo** [comment]
I am closing for now. Feel free to raise another issue if you run into more problems.
